webui/src/components/help.jsx

```diff
@@ -94,12 +99,13 @@ export const UserGuide = React.createClass({
 });
 
 export const TermsOfUse = React.createClass({
-    render() {
+	render: function() {
+		var termofuse;
         return (
             <div className="container-fluid">
                 <div className="row">
                     <div className="col-sm-10">
-                        <h1>Terms of Use</h1>
+						<Termofuse term_of_use={termofuse}/>
```

Comment:
Please use spaces for indenting and use 4-space tabs, so that the indentation is uniform (here and below)